### PR TITLE
Remove OIDC references/settings.

### PR DESCRIPTION
**Pull request by dianakhuang**
## Description

Remove the last bits of OIDC references/settings.

https://openedx.atlassian.net/browse/BOM-1225

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -23,50 +23,42 @@ If you need such overrides, create a file :file:`registrar/settings/private.py`.
...
-This service relies on the edX OIDC (`OpenID Connect`_) authentication provider for login. Note that OIDC is built atop
-OAuth 2.0, and this document may use the terms interchangeably. Under our current architecture the LMS serves as our
-authentication provider.
+Configure edX OAuth
+-------------------
+This service relies on the LMS server as the OAuth 2.0 authentication provider.
 
-Configuring Registrar service to work with OIDC requires registering a new client with the authentication
-provider and updating the Django settings for this project with the client credentials.
+Configuring {{ cookiecutter.project_name }} to communicate with other IDAs using OAuth requires registering a new client with the authentication
```
Comment (kdmccormick) [inline docs/getting_started.rst]: :cookie:

File: docs/getting_started.rst [hunk 2]
```diff
@@ -23,50 +23,42 @@ If you need such overrides, create a file :file:`registrar/settings/private.py`.
...
-A new OAuth 2.0 client can be created at ``http://127.0.0.1:8000/admin/oauth2/client/``.
-
-    1. Click the :guilabel:`Add client` button.
+A new OAuth 2.0 client can be created when using Devstack by visiting ``http://127.0.0.1:18000/admin/oauth2_provider/application/``.
+    1. Click the :guilabel:`Add Application` button.
     2. Leave the user field blank.
-    3. Specify the name of this service, ``Registrar service``, as the client name.
-    4. Set the :guilabel:`URL` to the root path of this service: ``http://localhost:8003/``.
-    5. Set the :guilabel:`Redirect URL` to the OIDC client endpoint: ``https://localhost:8003/complete/edx-oidc/``.
+    3. Specify the name of this service, ``{{ cookiecutter.project_name }}``, as the client name.
```
Comment (kdmccormick) [inline docs/getting_started.rst]: :cookie:

File: docs/getting_started.rst [hunk 3]
```diff
@@ -23,50 +23,42 @@ If you need such overrides, create a file :file:`registrar/settings/private.py`.
...
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
-| SOCIAL_AUTH_EDX_OIDC_URL_ROOT                       | OAuth 2.0 authentication URL                                               | http://127.0.0.1:8000/oauth2                                             |
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
-| SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY        | OIDC ID token decryption key. This value is used to validate the ID token. | (This should be the same value as SOCIAL_AUTH_EDX_OIDC_SECRET.)          |
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
+:file:`{{cookiecutter.repo_name}}/settings/local.py`). The table below describes the relevant settings.
```
Comment (kdmccormick) [inline docs/getting_started.rst]: :cookie:

**Review by kdmccormick — APPROVED** [review]
:tada: 

Also thanks for the doc updates! LGTM pending some missed copypasta.

**Comment by dianakhuang** [comment]
Thanks for catching those 🍪 s. 🤦‍♀

**Pull request merged**